lutris/sysoptions.py

```diff
@@ -29,6 +29,24 @@ def get_resolution_choices():
     return resolution_choices
 
 
+def get_locale_choices():
+    """Return list of available locales as label, value tuples
+    suitable for inclusion in drop-downs.
+    """
+    locales = system.get_locale_list()
+
+    # adds "(recommended)" string to utf8 locales
+    locales_humanized = locales.copy()
+    for index, locale in enumerate(locales_humanized):
+        if "utf8" in locale:
+            locales_humanized[index] += " " + _("(recommended)")
```

Comment:
if the utf8 is recommended, why do we give the option of choosing the non recommended one? Shouldn't we just drop them?